integration_tests/__tests__/multi_project_runner.test.js

```diff
@@ -154,6 +154,50 @@ test('"No tests found" message for projects', () => {
   );
 });
 
+test('objects in project configuration', () => {
+  writeFiles(DIR, {
+    '__tests__/file1.test.js': `
+      test('foo', () => {});
+    `,
+    '__tests__/file2.test.js': `
+      test('bar', () => {});
+    `,
+    'jest.config.js': `module.exports = {
+      projects: [
+        { testMatch: ['<rootDir>/__tests__/file1.test.js'] },
+        { testMatch: ['<rootDir>/__tests__/file2.test.js'] },
+      ]
+    };`,
+    'package.json': '{}',
+  });
+
+  const {stdout, stderr} = runJest(DIR);
+  expect(stderr).toContain('Test Suites: 2 passed, 2 total');
```

Comment:
I realize this probably follows the pattern of the other tests here, but could you use `extractSummary` from `utils` and use a snapshot?

Also, just a quick `status === 0` check would be nice

Reply:
Cool, will do.